src/services/statisticMail/index.js

```diff
@@ -0,0 +1,44 @@
+const { Configuration } = require('@schul-cloud/commons');
+const hooks = require('./hooks/index');
+const globalHooks = require('../../hooks');
+const { SC_SHORT_TITLE } = require('../../../config/globals');
+
+class StatisticMailService {
+	constructor(app) {
+		this.app = app;
+	}
+
+	create() {
+		return this.app.service('statistics').find().then((statData) => {
+			console.log(statData);
+			let htmlMailContent = `<h1>Statistik für ${SC_SHORT_TITLE}</h1>`;
+			htmlMailContent += `<table><tr><td>Schulen</td><td>${statData.schools}</td></tr>`;
+			htmlMailContent += `<tr><td>SchülerInnen</td><td>${statData.students}</td></tr>`;
+			htmlMailContent += `<tr><td>Lehrkräfte</td><td>${statData.teachers}</td></tr>`;
+			htmlMailContent += `<tr><td>Aufgaben</td><td>${statData.homework}</td></tr>`;
+			htmlMailContent += `<tr><td>Abgaben</td><td>${statData.submissions}</td></tr>`;
+			htmlMailContent += `<tr><td>Kurse</td><td>${statData.courses}/td></tr>`;
+			htmlMailContent += '</table>';
+			const mailService = this.app.service('mails');
+
+			if (Configuration.has('ADMIN_MAIL_RECEIVERS')) {
+				mailService.create({
+					email: Configuration.get('ADMIN_MAIL_RECEIVERS'),
+					subject: `Statistik für ${SC_SHORT_TITLE}`,
+					content: {
+						html: htmlMailContent,
+					},
+				});
+				return 'Mail sent';
```

Comment:
maybe i would put this in a trycatch with await mailService.create(...) , so you can be really sure that "Mail sent" is correct.